recipe/meta.yaml

```diff
@@ -1,58 +1,52 @@
 {% set name = "scikit-image" %}
-{% set version = "0.22.0" %}
-{% set sha256 = "2875c81ffb224f9f25a1274734b502fb993cc729a2c790009d38035e52e6a123" %}
-
-{% set build_number = "0" %}
+{% set version = "0.23.2" %}
 
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
-  url: https://github.com/scikit-image/scikit-image/archive/refs/tags/v{{ version }}.tar.gz
-  sha256: {{ sha256 }}
-  patches:
-    - skip-broken-tests.patch
+  url: https://github.com/scikit-image/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: be1b8d38fb51b8076d829f682646f9fdb61f9936e43a8b775fb594c7c7930a88
 
 build:
-  number: {{ build_number }}
-  skip: true  # [py<39]
+  number: 0
+  skip: true  # [py<310]
   entry_points:
     - skivi = skimage.scripts.skivi:main
 
-
 requirements:
   build:
     # pythran code needs clang-cl on windows
     - clang                 # [win]
-    - lld     # [win]
+    - lld                   # [win]
     - {{ compiler("c") }}
     - {{ compiler("cxx") }}
-    - m2-patch              # [win]
-    - patch                 # [not win]
   host:
     - python
-    - cython >=0.29.32,<3
+    - cython >=3.0.4
     - numpy {{ numpy }}
     - packaging
     - pip
     - pythran
     - meson-python
     - lazy_loader >=0.3
+    - setuptools
```

Comment:
is setuptools/wheel needed with a meson-python build backend? I can't see where these requirements got added upstream

Reply:
It was added by maintainers explicitly to the build section of pyproject.toml https://github.com/scikit-image/scikit-image/commit/09c7f1d43947acfdd9086bf689cdce340214774f with a note:
`Since it is mainly for developers and packagers and we don't test old versions, I think it makes sense to require a recent version.`

But I guess it's not needed.

Reply:
I removed `setuptools` and `wheel`